packages/figma-block/src/FigmaEmptyBlock.tsx

```diff
@@ -12,13 +12,15 @@ export const FigmaEmptyBlock = ({ onOpenAssetChooser }: Props): ReactElement =>
     return (
         <button
             data-test-id="figma-empty-block"
-            className="tw-group tw-w-full tw-py-16 tw-px-4 tw-border-dashed tw-border tw-cursor-pointer tw-text-center tw-border-line-x-strong hover:tw-border-black"
+            className="tw-group tw-w-full tw-py-16 tw-px-4 tw-border-dashed tw-border tw-cursor-pointer tw-text-center tw-border-line-x-strong hover:tw-border-[color-mix(in_srgb,_var(--f-theme-settings-body-color)_70%,_transparent)]"
```

Comment:
instead of `tw-border-line-x-strong` use 70% body color.

And on hover it should be 100% body color.